konig-core/src/main/java/io/konig/shacl/PropertyConstraint.java

```diff
@@ -529,5 +530,12 @@ public URI getTermStatus() {
 
 	public void setTermStatus(URI termStatus) {
 		this.termStatus  = termStatus ;
+	}
+	public boolean isRequired() {
+		return required;
```

Comment:
return minCount!=null && minCount>1

However, I think you might be interested in a different method which asserts that the property is single-valued and required.  If that's the case, then you should add the following method...

```
boolean isRequiredSingleValue() {
  return minCount!=null && minCount==1 && maxCount!=null && maxCount==1;
}
```